client/landing/stepper/declarative-flow/internals/steps-repository/__user/style.scss

```diff
@@ -49,6 +49,16 @@
 					line-height: 3.25;
 				}
 			}
+			.formatted-header__title:has(+ p) {
+				@include break-small {
+					line-height: 1.9;
+				}
+			}
+			.formatted-header__subtitle {
+				@include break-small {
+					padding-bottom: 1em;
+				}
+			}
```

Comment:
We need to test this in other flows to ensure it won't be a problem.

Reply:
Yep this is working fine in other flows too.

I know we don't need to worry about mobile layout either, but I gave it a look and it seems fine too.